src/components/AuthForm/LoginForm.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useEffect, useState } from "react";
+import FormInput from "./FormInput";
+import FormButton from "@/components/AuthForm/FormButton";
+import Link from "next/link";
+import { useForm } from "react-hook-form";
+import { useRouter } from "next/router";
+import { useAuthStore } from "@/stores/authStore";
+import { loginAPI } from "@/apis/authAPI";
+
+const LoginForm = () => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    mode: "onBlur",
+  });
+
+  const router = useRouter();
+  const [isLoading, setIsLoading] = useState(false);
+  const [mounted, setMounted] = useState(false);
+  const { login, isInitialized } = useAuthStore();
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
+
+  const onSubmit = async (data: { email: string; password: string }) => {
+    if (!mounted || !isInitialized) return;
```

Comment:
취향은 맞지만 보통 대부분이 아래와 같은 방향을 권장해요.
```suggestion
    if (!mounted || !isInitialized) {
     return;
    }
```